services/external-secrets/0.17.0/release/release.yaml

```diff
@@ -0,0 +1,32 @@
+---
+apiVersion: helm.toolkit.fluxcd.io/v2
+kind: HelmRelease
+metadata:
+  name: external-secrets
+  namespace: ${releaseNamespace}
+spec:
+  chart:
+    spec:
+      chart: external-secrets
+      sourceRef:
+        kind: HelmRepository
+        name: charts.external-secrets.io
```

Comment:
They don't host in OCI I suppose?

Reply:
They do publish charts to ghcr, we can easily switch when our tooling supports that.